examples/jsonExample/src/main/java/io/realm/examples/json/City.java

```diff
@@ -19,24 +19,6 @@
 import io.realm.RealmObject;
 
 public class City extends RealmObject {
-
-    private String name;
-    private long votes;
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public long getVotes() {
-        return votes;
-    }
-
-    public void setVotes(long votes) {
-        this.votes = votes;
-    }
-
+    public String name;
+    public long votes;
```

Comment:
Would it make more sense to move these to the introExample + add custom methods there? Then we introduce this feature right away instead of hiding it here.